### PR TITLE
Add region restrictions to the prerequisites

### PR DESCRIPTION
**Pull request by guangyee**

### CODE REVIEW

**Comment by yarunachalam** [comment]
LGTM

**Review by yarunachalam — APPROVED** [review]
LGTM

File: content/rancher-prime/azure/contents.lr [hunk 1]
```diff
@@ -8,7 +8,9 @@ body:
 
 1. A Rancher-compatible AKS cluster. Please see
    [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-6/)
-   for more details.
+   for more details. Also, the Rancher Prime PAYG offering can only be installed onto clusters in regions where both the
```
Comment (jeremy-moffitt) [inline content/rancher-prime/azure/contents.lr]: I would ditch `Also, `.... and just start with "The Rancher Prime"

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
nitpick, but lets ditch the Also

File: content/rancher-prime/azure/contents.lr (hunk 1 above)
Comment (guangyee) [reply content/rancher-prime/azure/contents.lr]: Done

**Review by jeremy-moffitt — APPROVED** [review]

**Pull request merged**